### ISSUE TITLE
Image short code missing alt text for accessibility

### ISSUE THREAD

**Issue by timcappalli** [gethinode/hinode#1331]
## Problem or enhancement idea

The markdown native image syntax uses the text in brackets as the alt text for accessibility. The image short code does not have a parameter to provide alt text.

## Proposed solution

Add a a new "alt" parameter to the image short code and add provide in the image tag when populated.

**Comment by timcappalli** [comment]
Looks like `title` populates alt.
